htdocs/fourn/commande/card.php

```diff
@@ -1057,7 +1057,7 @@
 
 		if ($cancel) {
 			$action = '';
-		} elseif ($methodecommande <= 0) {
+		} elseif ($methodecommande <= 0 && !dolGetGlobalInt('SUPPLIER_ORDER_MODE_OPTIONAL')) {
```

Comment:
The function dolGetGlobalInt does not exist

Reply:
ok @eldy, it is fixed now